conans/client/generators/virtualenv.py

```diff
@@ -182,9 +185,23 @@ def _files(self, flavor, activate_tpl, deactivate_tpl, environment_filename):
         deactivate_content = deactivate_tpl.render(modified_vars=modified_vars, new_vars=new_vars)
 
         environment_lines = ["{}={}".format(name, value) for name, value, _ in ret]
+        # This blank line is important, otherwise the script doens't process last line
         environment_lines.append('')
 
-        return activate_content, deactivate_content, os.linesep.join(environment_lines)
+        if platform.system() == "Windows":
```

Comment:
Up to my understanding (https://github.com/conan-io/conan/issues/6660#issuecomment-598198641), if Conan is installed inside Cygwin the `platform.system()` will return `CYGWIN-xxxx` so this `if` will prevent half the fix we are trying to implement in this PR.

Would it be ok to replace always CRLF -> LF for `VirtualEnvGenerator. SH_FLAVOR ` without worrying about the platform?

Reply:
> Would it be ok to replace always CRLF -> LF for VirtualEnvGenerator. SH_FLAVOR without worrying about the platform?

Good point. I tried to not change more behavior than intended, and the ``platform.system()`` was embedded in the ``normalize()`` function. Changed.